src/components/heroes.json

```diff
@@ -0,0 +1,102 @@
+[
```

Comment:
- I don't get why you need this file
- It's not a component, so it should not be in `src/components`